webapi/webapi-webgl-khronos-tests/webgl/khronos/resources/js-test-pre.js

```diff
@@ -140,6 +168,14 @@ function escapeHTML(text)
 
 function testPassed(msg)
 {
+    if (msg !== "successfullyParsed is true") {
```

Comment:
It's only for WTS. To save subcases' results into result json file, WTS uses document titile suffixing with '/' and index as the subcase's name. If the judgment statement "if (msg !== "successfullyParsed is true") " satisfied, the result of the whole test page is BLOCK.